cmd/utils/flags.go

```diff
@@ -182,6 +182,10 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
+	BlockHashWhitelistsFlag = cli.StringSliceFlag{
+		Name:  "whitelist",
+		Usage: "Reject peers which send non-matching block hashes (<block number>=<hash>)",
```

Comment:
I think the description would be clearer as:

`Comma separated block number-to-hash mappings to enforce (<number>=<hash>)`

I don't think we need to mention peers and networking in this code. That's an implementation detail if you use the flag correctly.

Reply:
Updated in b406a1c57 as well.